### ISSUE TITLE
Are users topping up their accounts in time?

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#331]
When I explained in Lille how Liberapay doesn't charge users automatically but simply notifies them that they need to top up their accounts, one question that came up was: does that actually work? Do users top up their accounts when they're notified? I couldn't answer, because we don't keep track of that. We should make some stats on that.

**Comment by SimonSarazin** [comment]
Why not give the possibility to charge automtically the money once it get low, as it was with gratipay ? I think we lose lot's of people by asking people to connect and charge again once they don't get anymore money. For example, with framasoft, i decided to give 5€ euro by month and i like it as i don't have anything to do. But if i had to go again on the website each 5 o 10 months, sure that i could stop to give to them, juste because i don't take the time to activate again my donation....

**Comment by Changaco** [comment]
The reason why we don't charge automatically on payday like gratipay is that we can't do it if the amount is greater than 100 euros, and that we have to charge a card a first time soon after the user inputs the number otherwise it expires, which is why I ended up removing the automatic charge logic entirely to simplify payday.

We could bring back automatic charges for amounts below 100 euros, or we could focus on direct debits first (#57).

We also need to acknowledge that not everyone likes automatic charges, so we should do a better job at getting people to top up their accounts in time even if we add an automatic debit option. That means sending multiple emails, because a single email a week before isn't ideal for many people, and adding a one-click link in these emails so that adding money requires less effort from the user.

**Comment by trebmuh** [comment]
I do agree with the 2 reasonings here. Some people like being asked before being charged again on their account, some like being charged automatically when their account provision are getting low.

Would it then be possible to have a checkbox when someone decide to donate to a project/person which would ask this very question: when you account provision is getting low, would you like:
1) to be re-asked for recharging ?
2) to be automatically charged ?
3) this donation to be stopped ?
Then the donator would have the choice for each project he is giving to.

**Comment by Changaco** [comment]
Yes, that's pretty much what we're heading towards. Note that only some payment methods can be used for automatic charges, so that choice has side-effects.

**Comment by trebmuh** [comment]
Nice :+1:

**Comment by jelv** [comment]
+1 for automatic account fill up from https://mastodon.social/@kurtm/99174108834254457
> 1) Have to fill up account by hand.
